boefjes/boefjes/plugins/kat_servicebanner/main.py

```diff
@@ -0,0 +1,40 @@
+import socket
+
+from boefjes.job_models import BoefjeMeta
+
+TIMEOUT = 1.0
+
+
+def get_ssh_sock(ip, port, timeout):
+    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    sock.settimeout(timeout)
+    try:
+        sock.connect((ip, port))
+        return sock
+    except Exception:
+        return None
+
+
+def get_ssh_banner(sock):
+    try:
+        banner = sock.recv(1024)
+        try:
+            banner = banner.decode().strip()
+        except UnicodeDecodeError:
+            banner = banner.decode("latin1").strip()
+        sock.close()
+        return banner
+    except Exception as e:
```

Comment:
ReRaise this error, or dont catch it here, catch it around line 38 and return a different output-set if we find an error instead of a banner.